exporter/core/constants.py

```diff
@@ -104,3 +104,6 @@ class NotificationType:
 class LocationType:
     SEA_BASED = "sea_based"
     LAND_BASED = "land_based"
+
+
+PRODUCT_CATEGORY_FIREARM = "group2_firearms"
```

Comment:
maybe make the entire `ItemType` enum here
```suggestion
class ItemCategory:
    GROUP1_PLATFORM = "group1_platform"
    GROUP1_DEVICE = "group1_device"
    GROUP1_COMPONENTS = "group1_components"
    GROUP1_MATERIALS = "group1_materials"
    GROUP2_FIREARMS = "group2_firearms"
    GROUP3_SOFTWARE = "group3_software"
    GROUP3_TECHNOLOGY = "group3_technology"
```

Reply:
good idea - but once the `firearms` branch has everything merged in otherwise I predicts more conflicts